src/main/java/com/dabsquared/gitlabjenkins/gitlab/api/impl/ResteasyGitLabClientBuilder.java

```diff
@@ -53,7 +53,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import static java.net.Proxy.NO_PROXY;
+import static java.net.Proxy.Type.HTTP;
```

Comment:
Is it possible to instead rely on `java.net.Proxy.*` we could use the [Async Http Client Plugin](https://wiki.jenkins.io/display/JENKINS/Async+Http+Client+Plugin) (which IIRC has the proxy configuration in the UI under Manage Jenkins > Manage Plugins > Advanced tab)?

Reply:
The UI unter Manage Plugins is the one built into Jenkins itself. ResteasyGitLabClientBuilder uses them already. What i could do is to use Proxy.Type.DIRECT instead of Proxy.Type.HTTP in the check to allow SOCKS-proxied connections but then we'd have to create a socks-configuration further down. As of relying on the Async Http Client Plugin - i'm not sure whether it is possible to use this with Resteasy